render-only-sample/rosumd/RosUmdResource.cpp

```diff
@@ -13,12 +13,13 @@
 #include "RosContext.h"
 
 #include "Vc4Hw.h"
+#include <memory>
```

Comment:
It will be added during merge - because my branch is so old, that is has no precomp.h in UMD added yet.